### PR TITLE
Merge register & sampler v0.1

### LINKED ISSUE #18: Join Sampler into Temporal
Merge current Sampler into Temporal structure

### PR DESCRIPTION
**Pull request by AguirreNicolas**
* Fix logger.
* increase times.
* SQL query fixes.
* add Temporal `ApplicationError`.
* Move files into `activities/lmeh` and `activities/helm` structure.
* Docker examples for Register and Sampler.
* Readme clean

Close #18

### CODE REVIEW

File: docker-compose/apps/docker-compose.yaml
Comment (RawthiL) [inline docker-compose/apps/docker-compose.yaml:1]: Why is this file here and not in infrastructure?

**Review by RawthiL — COMMENTED** [review]
Some files seem to be out of place and should be merged with dev infrastructure.

**Review by RawthiL — DISMISSED** [review]

**Review by RawthiL — APPROVED** [review]

**Review by jorgecuesta — APPROVED** [review]
